arangod/IResearch/IResearchFilterFactory.cpp

```diff
@@ -182,29 +183,41 @@ Result getLatLong(ScopedAqlValue const& value, S2LatLng& point,
         return error::failedToEvaluate(funcName, argIdx);
       }
 
-      point = S2LatLng::FromDegrees(lat, lon).Normalized();
-      return {};
-    }
+      auto latLng = S2LatLng::FromDegrees(lat, lon).Normalized();
+      if (options.stored != StoredType::S2Centroid &&
```

Comment:
#ifdef USE_ENTERPRISE ?

Reply:
Why?

Reply:
In general I believe commonly we don't need to add `#ifdef` if code compiles without it